libs/domain/picking/src/components/picking-lists/picking-lists.styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css } from 'lit';
+
+export const styles = css`
+  :host {
+    padding: 24px 16px 0;
+    width: 100%;
+    display: grid;
+    gap: 34px;
+  }
+
+  oryx-modal [slot='heading'] {
+    font-size: 18px;
+    font-weight: 600;
+  }
+
+  oryx-modal oryx-button {
+    width: 100%;
+  }
```

Comment:
It looks like a separate task that is not related to migrating picking lists to picking domain. I guess we can create a new subtask in the current story to avoid blocking Alvina and Pavlo PRs with this PR